packages/admin-next/dashboard/src/routes/products/product-organization/components/product-organization-form/product-organization-form.tsx

```diff
@@ -23,47 +22,85 @@ type ProductOrganizationFormProps = {
 const ProductOrganizationSchema = zod.object({
   type_id: zod.string().optional(),
   collection_id: zod.string().optional(),
-  category_ids: zod.array(zod.string()).optional(),
-  tags: zod.array(zod.string()).optional(),
+  category_ids: zod.array(zod.string()),
+  tag_ids: zod.array(zod.string()),
 })
 
 export const ProductOrganizationForm = ({
   product,
 }: ProductOrganizationFormProps) => {
   const { t } = useTranslation()
   const { handleSuccess } = useRouteModal()
-  const { product_types, isLoading: isLoadingTypes } = useProductTypes()
-  const { product_tags, isLoading: isLoadingTags } = useTags()
-  const { collections, isLoading: isLoadingCollections } = useCollections()
-  const { product_categories, isLoading: isLoadingCategories } = useCategories()
+
+  const collections = useComboboxData({
+    queryKey: ["product_collections"],
+    queryFn: sdk.admin.collection.list,
+    getOptions: (data) =>
+      data.collections.map((collection) => ({
+        label: collection.title!,
+        value: collection.id!,
+      })),
+  })
+
+  const types = useComboboxData({
+    queryKey: ["product_types"],
+    queryFn: client.productTypes.list,
+    getOptions: (data) =>
+      data.product_types.map((type) => ({
+        label: type.value,
+        value: type.id,
+      })),
+  })
+
+  const tags = useComboboxData({
+    queryKey: ["product_tags"],
+    queryFn: client.tags.list,
+    getOptions: (data) =>
+      data.tags.map((tag) => ({
```

Comment:
todo: when trying to edit organization section `data.tags` is undefined and this throws error

Reply:
I can't get my local project to run when I link Medusa, so I don't have the new endpoint to test with. Can you try again with the latest changes? 😄

Reply:
Works now 👍